app/views/welcome/index.html.erb

```diff
@@ -4,7 +4,7 @@
 <div class="wrapper-shadow">
   <%= render partial: "hero" %>
   <%= render partial: "about" %>
-  <%= render partial: "widget_sample" %>
+  <%#= render partial: "widget_sample" %>
```

Comment:
I didn't know if we wanted to keep this or not. If we use the non-heroku URL it shows up so we can keep it if we want.